src/composition/supergraph/config/resolve/mod.rs

```diff
@@ -193,6 +192,39 @@ impl LazilyResolvedSupergraphConfig {
             Err(errors)
         }
     }
+
+    /// Fully resolves a [`LazilyResolvedSupergraphConfig`] into a [`FullyResolvedSubgraphs`]
+    /// by retrieving all the schemas as strings
+    pub async fn fully_resolve_subgraphs(
```

Comment:
worth calling out that this is a one-way conversion because information gets lost? Might help a future dev know whether this is the right thing to reach for